tests/pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>io.jenkins.jenkinsfile-runner</groupId>
     <artifactId>parent</artifactId>
-    <version>1.0-beta-12-SNAPSHOT</version>
+    <version>1.0-beta-18-SNAPSHOT</version>
```

Comment:
🤷‍♂️

Reply:
Yeah, that was from when I was trying to get the its to work. =)